### ISSUE TITLE
Make it more obvious when Refinery can't communicate to Honeycomb

### ISSUE THREAD

**Issue by cartermp** [honeycombio/refinery#812]
For example, let's say the Honeycomb API is completely unreachable (like in a recent outage we had). Refinery could more prominently highlight, "Hey, I'm getting 500s from my upstream, and it's not you - it's Honeycomb".

Today, you can set up Metrics to tell you this. But not everyone running Honeycomb will do that, nor will they be looking at Refinery metrics.

Is there some other notification or...something that could be done that's a bit more obvious for this use case?

**Comment by kentquirk** [comment]
One possibility would be to create backpressure -- if refinery goes a certain time without being able to send anything upstream, instead of continuing to accept data, it could start returning, say, a `503 - Service Unavailable`. The body of the response could include an explanation that this is due to an upstream problem and a link to the Honeycomb status page.

Does that sound plausible?

**Comment by cartermp** [comment]
Mmm, that could probably work I think.

**Comment by kentquirk** [comment]
This is going on hold until [this](https://github.com/honeycombio/hound/pull/15962) resolves since it's going to make some modifications to Husky, and this work will want to use those.

**Comment by VinozzZ** [comment]
This [husky PR](https://github.com/honeycombio/husky/pull/219) should unblock this PR

**Comment by kentquirk** [comment]
We don't have time to fix and reliably test this in 2.3, so I'm moving it to 2.4.
